pkg/parsers/filters/parse.go

```diff
@@ -36,9 +37,10 @@ func ParseFlag(arg string, prev Args) (Args, error) {
 	return filters, nil
 }
 
+// ErrorBadFormat bad format for filter
 var ErrorBadFormat = errors.New("bad format of filter (expected name=value)")
 
-// packs the Args into an string for easy transport from client to server
+// ToParam packs the Args into an string for easy transport from client to server
```

Comment:
I'm not sure it should be plural, as it returns only one string, don't you think ? :)